app/components/Edit/EditSections.js

```diff
@@ -328,6 +329,24 @@ class EditSections extends React.Component {
 
   }
 
+  handleDeactivation(type, id) {
+    let path = null;
+    if (type === 'resource') {
+      path = `/api/resources/${id}`;
+    } else if (type === 'service') {
+      path = `/api/services/${id}`;
+    }
+    dataService.APIDelete(path, { change_request: { status: "2" } })
+    .then(() => {
+      alert('Successfully deactivated! \n \nIf this was a mistake, please let someone from the ShelterTech team know.')
```

Comment:
Was there a reason that you chose alert instead of confirm here? It would prevent accidental clicks.